docs/src/examples.md

```diff
@@ -37,6 +37,51 @@ nothing # hide
 
 ![Cooke triplet visualization](assets/cooke.png)
 
+## Zoom Lens
+
+```@example
+using OpticSim
+
+using DataFrames
+function zoom_lens(pos = 1)
+    if pos == 0
+        stop = 2.89
+        zoom = 9.48
+        dist = 4.46970613
+    elseif pos == 1
+        stop = 3.99
+        zoom = 4.48
+        dist = 21.21
+    else
+        stop = 4.90
+        zoom = 2.00
+        dist = 43.81
+    end
+    return AxisymmetricOpticalSystem{Float64}(
+        DataFrame(Surface = [:Object, :Stop, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, :Image],
+                  Radius = [Inf64, Inf64, -1.6202203499676E+01, -4.8875855327468E+01, 1.5666614444619E+01, -4.2955326460481E+01, 1.0869565217391E+02, 2.3623907394283E+01, -1.6059097478722E+01, -4.2553191489362E+02, -3.5435861091425E+01, -1.4146272457208E+01, -2.5125628140704E+02, -2.2502250225023E+01, -1.0583130489999E+01, -4.4444444444444E+01, Inf64],
+                  Aspherics = [missing, missing, missing, missing, missing, [(4, 1.03860000000E-04), (6, 1.42090000000E-07), (8, -8.84950000000E-09), (10, 1.24770000000E-10), (12, -1.03670000000E-12), (14, 3.65560000000E-15)], missing, missing, [(4, 4.27210000000E-05), (6, 1.24840000000E-07), (8, 9.70790000000E-09), (10, -1.84440000000E-10), (12, 1.86440000000E-12), (14, -7.79750000000E-15)], [(4, 1.13390000000E-04), (6, 4.81650000000E-07), (8, 1.87780000000E-08), (10, -5.75710000000E-10), (12, 8.99940000000E-12), (14, -4.67680000000E-14)], missing, missing, missing, missing, missing, missing, missing],
+                  Thickness = [Inf64, 0.0, 5.18, 0.10, 4.40, 0.16, 1.0, 4.96, zoom, 4.04, 1.35, 1.0, 2.80, 3.0, 1.22, dist, missing],
+                  Material = [OpticSim.GlassCat.Air, OpticSim.GlassCat.Air, OpticSim.GlassCat.OHARA.S_LAH66, OpticSim.GlassCat.Air, OpticSim.GlassCat.NIKON.LLF6, OpticSim.GlassCat.Air, OpticSim.GlassCat.OHARA.S_TIH6, OpticSim.GlassCat.OHARA.S_FSL5, OpticSim.GlassCat.Air, OpticSim.GlassCat.OHARA.S_FSL5, OpticSim.GlassCat.Air, OpticSim.GlassCat.OHARA.S_LAL8, OpticSim.GlassCat.OHARA.S_FSL5, OpticSim.GlassCat.Air, OpticSim.GlassCat.OHARA.S_LAH66, OpticSim.GlassCat.Air, missing],
```

Comment:
I changed OHARA.FSL5 -> OHARA.S_FSL5 because none of our default glass catalogs contain the former. Does anyone know if this is innacurate?

Reply:
Comparing to previous versions of the example, I can see that we also changed HIKARI.LLF6 to NIKON.LLF6, as well as a few other changes similar to the one I mentioned above. I don't know anything about glass cat naming conventions so it'd be good if someone could confirm that this example is OK.

Reply:
https://microsoft.github.io/OpticSim.jl/previews/PR108/examples/#Zoom-Lens

Reply:
> I can see that we also changed HIKARI.LLF6 to NIKON.LLF6, as well as a few other changes similar to the one I mentioned above. I don't know anything about glass cat naming conventions

The same material will often be manufactured by a number of companies, the optical properties should be (near) identical, the secondary properties might vary a bit. The results should be near identical, certainly not different enough to matter for an example.

> I changed OHARA.FSL5 to OHARA.S_FSL5 because none of our default glass catalogs contain the former

Companies also come up with new versions of the same material which again have (near) identical optical properties but different secondary properties (cost etc). It should also be safe to substitute these (e.g. BK7 and N_BK7 from SCHOTT) - I'm assuming this is a similar case, but I'm not sure - you can check what differences there are between by looking at the AGF or output of info().

Reply:
The zoom lens example is not right - so maybe my assumption about S_FSL5 is wrong - or something else is broken...

Reply:
I'll mention it to Brian tonight at our meeting. If we still need to re-optimise the example, I'll take it off this PR so we can merge the important changes for now.

Reply:
We'll keep it up for now - PR should be ready for merging, if you're happy with it!